chutney/server-core/src/main/java/com/chutneytesting/server/core/domain/scenario/campaign/CampaignExecution.java

```diff
@@ -124,7 +125,7 @@ public CampaignExecution(Long executionId,
         }
     }
 
-    public void initExecution(List<TestCaseDataset> testCaseDatasets, String executionEnvironment, String userId) {
+    public void initExecution(List<TestCaseDataset> testCaseDatasets, String executionEnvironment, String userId, List<String> tags) {
```

Comment:
For initExecution(...) and startScenarioExecution(...), the userId parameter seems not useful as it's always the same as the constructor one and it would be strange to change execution user while executing a campaign.

Just remove the userId parameter.